### ISSUE TITLE
Missing end quote for quoted column can cause Tokenizer to hang

### ISSUE THREAD

**Issue by davidrwinters** [super-csv/super-csv#37]
We are using super-csv to parse rather huge CSV files in an import process where some of the files have 100's of millions of rows.  Often times the CSV files are not well formatted and will have unescaped quotes in them, which causes the Tokenizer to scan for the ending quote for a very long time.  Probably if we would let it run long enough the JVM would run out of heap.  In this case of missing end quotes, we would like the Tokenizer to fail fast when a maximum number of rows in the quoted column has been reached.  I have forked your repo and added this functionality.  I can make a pull request if you would like the changes.  I also added some unit tests to test this functionality.

Unfortunately, I cannot provide any data files since they are huge and contain private data.

**Comment by jamesbassett** [comment]
Hi David, I believe this is a duplicate of #3 which has been addressed by PR #34 (which I was planning on merging this weekend as I just got back from overseas). I appreciate the work you've put in (and I don't want to discourage you from contributing!) but it looks almost identical to the other PR, so I'm going to close this as a duplicate. I recommend that you check the open issues and pull requests to save yourself from this in the future. Happy coding :)
